style.css

```diff
@@ -0,0 +1,65 @@
+html, body, main {
```

Comment:
Please, put each different CSS selector into a new line. This would avoid linter errors.